js/App.js

```diff
@@ -0,0 +1,247 @@
+const controlledApp = {
+  name: "Car Race App",
+  description: "Canvas app for basic shapes controlling",
+  version: "1.0.0",
+  license: undefined,
+  author: "Lara Lorenzo",
+  canvasTag: undefined,
+  ctx: undefined,
+  frames: 0,
+  road: undefined,
+  car: undefined,
+  obstacle: [],
+  gameOver: false,
+  keys: {
+    left: 37,
+    right: 39,
+  },
+
+  canvasSize: {
+    w: 500,
+    h: 700,
+  },
+  init(id) {
+    console.log(window);
+    this.canvasTag = document.getElementById(id);
+    this.ctx = this.canvasTag.getContext("2d");
+    this.setDimensions();
+    this.createRoad();
+    this.createCar();
+    this.drawAll();
+    this.setEventListeners();
+
+    console.log(this.ctx);
+  },
+
+  setDimensions() {
+    this.canvasSize = {
+      w: window.innerWidth * 4,
+      h: window.innerHeight,
+    };
+    this.canvasTag.setAttribute("width", this.canvasSize.w);
+    this.canvasTag.setAttribute("height", this.canvasSize.h);
+  },
+
+  createRoad() {
+    this.road = new Road(this.ctx, 0, 0, 500, 700, "road.png");
+  },
+
+  createCar() {
+    this.car = new Car(this.ctx, 200, 200, 50, 100, "car.png");
+  },
+
+  setEventListeners() {
+    document.onkeydown = (e) => {
+      e.keyCode === this.keys.left ? this.ball.move("left") : null;
```

Comment:
Estos eventos tendrían que ser this.car.move, en lugar de ball, para que puedas mover el coche